compiler/x/codegen/UnaryEvaluator.cpp

```diff
@@ -45,9 +45,9 @@ TR::Register *OMR::X86::TreeEvaluator::unaryVectorArithmeticEvaluator(TR::Node *
    TR::InstOpCode::Mnemonic regRegOpcode;
    TR::InstOpCode::Mnemonic regMemOpcode;
 
-   switch (node->getOpCodeValue())
+   switch (node->getOpCode().getVectorOperation())
       {
-      case TR::vdsqrt:
+      case TR::vsqrt:
          regRegOpcode = supportsAvx ? OMR::InstOpCode::VSQRTPDRegReg : OMR::InstOpCode::SQRTPDRegReg;
```

Comment:
Assert that the element type is a double for now. Other types can be added later.

Reply:
Added an ASSERT.